gradle/libs.versions.toml

```diff
@@ -0,0 +1,105 @@
+[versions]
+kotlin-version = "1.6.10"
```

Comment:
does it make sense to store `kotlin` version and `kotlin` dependencies here, as they are overridable by in root `build.gradle` file?
Looks like it can cause unpredictable issues

Reply:
I'm going to drop version overriding in `1.6.*` branch